versions.json

```diff
@@ -411,16 +413,17 @@
         "commitish": "v1.9.2",
         "version_string": "1.9.2",
         "license": "MIT",
-        "consortium_build_number": "0",
+        "consortium_build_number": "2",
```

Comment:
why jumping to 2?

Reply:
incremented twice in two separate commits

Reply:
ah, i see - but we haven't actually pushed/published since 0 - so I think we can just go to 1 for now?